packages/firebase_database/lib/ui/firebase_animated_list.dart

```diff
@@ -207,6 +212,9 @@ class FirebaseAnimatedListState extends State<FirebaseAnimatedList> {
     if (!_loaded) {
       return widget.defaultChild ?? new Container();
     }
+    if (_model.isEmpty) {
+      return widget.emptyStateChild ?? new Container();
+    }
```

Comment:
won't this break the animation of the last item being removed?

Reply:
I am trying use this state 'this.emptyStateChild,' on my list to show something when the list is empty and is not working, is saying that isn't defined. How can i do to use this item?

Please sorry if i am in a wrong place.